.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,78 @@
+name: Bug Report
+description: This template is for reporting bugs found in the REMIND model
+title: "[Bug]: "
+labels: ["bug", "remind"]
+assignees:
+  - whitehacker
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we contact you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Please confirm the following
+      description: Please check the relevant boxes.
+      options:
+        - label: I have access to the PIK cluster
+        - label: I have access to the Input Data
+        - label: I have searched the existing issues
+        - label: I have provided detailed information about the bug
+        - label: I have provided steps to reproduce the issue
+        - label: I have provided information about my environment (OS)
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also, tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of REMIND are you running?
+      options:
+        - 3.2.0 (Latest)
+        - 3.1.0
+        - 3.0.0
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: What Operating System are you seeing the problem on?
+      multiple: true
+      options:
+        - Windows
+        - Linux
+        - macOS
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: checkboxes
+    id: terms
+    attributes:
```

Comment:
maybe we should add here that we are happy about any bug report and will try to fix bugs in REMIND soon. But that we con only provide very limited support for setting up and running REMIND outside PIK environment.

Reply:
Sure, and added.